src/js/AccessibilityUtils.js

```diff
@@ -654,7 +654,7 @@ axs.utils.getRoles = function(element, implicit) {
             roleObject.valid = result.valid = true;
         } else {
             roleObject.valid = false;
-            
+
```

Comment:
Weird that there was a bunch of spaces here before; can we remove the empty line as well?